draft-cenzano-moq-media-interop.md

```diff
@@ -184,19 +167,19 @@ should error with “Protocol violation”.
 Any change in encoding parameters MUST send a new AVCDecoderConfigurationRecord
 in Metadata
 
+
 ## Audio Object Format
 
 ~~~
 {
   Media Type (i)
   Seq ID (i)
-  Presence Flags (i)
   PTS Timestamp (i)
-  Duration (i)
   Timebase (i)
   Sample Freq (i)
   Num Channels (i)
-  Wall Clocl (i)
+  Duration (i)
+  Wall Clock (i)
```

Comment:
I have a slight pref for keeping the common fields in the same spots between audio an video, but we can shuffle it later.